app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] As required for the project, kindly make sure your application asks for confirmation of email. That means the `:confirmable` should be also enabled in the user model. It should be something similar to what is below:
```ruby
	devise :database_authenticatable, :registerable,
	            :recoverable, :rememberable, :validatable, :confirmable
```